sdk/constants/futures.ts

```diff
@@ -2,10 +2,10 @@ import { gql } from 'graphql-request';
 
 import { FuturesMarketAsset, FuturesMarketConfig, FuturesMarketKey } from 'sdk/types/futures';
 
-export const FUTURES_ENDPOINT_OP_MAINNET = `https://subgraph.satsuma-prod.com/${process.env.NEXT_PUBLIC_SATSUMA_API_KEY}/kwenta/optimism-perps/api`;
+export const FUTURES_ENDPOINT_OP_MAINNET = `https://api.thegraph.com/subgraphs/name/tburm/optimism-perps`;
```

Comment:
Would be nice to just remove these duplicated constants while we're making changes here

Reply:
I went through and deleted all the stuff that isn't used from this file

Reply:
Or transitioned into the SDK to avoid importing from this file.